coalib/results/TextPosition.py

```diff
@@ -32,6 +32,54 @@ def line(self):
     def column(self):
         return self._column
 
+    def __lt__(self, other):
+        """
+        Test whether ``self`` is behind the other
+        ``TextPosition``.
+
+        If the column in a ``TextPosition`` is ``None``, consider
+        whole line. If the line in a ``TextPosition`` is ``None``,
+        consider whole file.
+
+        :param other: ``TextPosition`` to compare with.
+        :return:      Whether this ``TextPosition`` is behind the other
+                      one.
+        """
+        if self.line is None or other.line is None:
+            return False
+        if self.line == other.line:
+            return (False
```

Comment:
noo, I meant the huge return statement - that logic requires more time to understand